earthly/docs/services/docker-compose.sitedocs.yml

```diff
@@ -0,0 +1,12 @@
+version: "3"
```

Comment:
I'm wondering if the distinction is really necessary here? In what use case would someone like to view the docs without live reloading? You can emulate this by using `serve` and just _not_ touching the files. 

I say this as I already had this exact thought and chose not to go with it for the above reason.

Reply:
Its 100% necessary.
I already have examples where live docs do not look the same as built docs.
Some plugins generate files and references to them very poorly.  Like they use relative paths in live docs, or worse, paths relative to the URL of the live server.
And in built docs, its a whole other path relative to the supposed URL you will deploy it on.  or other ways.  There doesn't;t seem to be any consistency.
Being able to check both will ensure I make docs which not only look ok to me when I write them, but won't be a broken mess when we deploy them.
This is a critical feature for me.
Finding some of the reasons why this happens is also on my TODO list, but its much lower importance to a lot of other things, so initially just knowing what I wrote is broken means I can quickly work around it and make working docs.

Reply:
Interesting, I wrongly assumed they would be the same 😅 You would think that `serve` would hot-compile the same static site that `build` does...either way, makes sense to have both until we can figure out what's going on.